.github/workflows/build.yml

```diff
@@ -50,3 +61,13 @@ jobs:
         run: |
           source poky/oe-init-build-env build
           bitbake platsch
+      - name: Cache Data
+        env:
+          CACHE_KEY: ${{ secrets.YOCTO_CACHE_KEY }}
+        if: ${{ env.CACHE_KEY }}
+        run: |
+          mkdir -p ~/.ssh
+          echo "$CACHE_KEY" >> ~/.ssh/id_ed25519
+          chmod 600 ~/.ssh/id_ed25519
+          rsync -rvx --ignore-existing build/downloads yocto-cache: || true
+          rsync -rvx --ignore-existing build/sstate-cache yocto-cache: || true
```

Comment:
Maybe use [`continue-on-error`](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error) for this step instead of `|| true`.
That would show the step as failing in the step overview.

Reply:
Hmm, I see this as an internal optimization of the build which shouldn't disturb people interested in the build itself.

Reply:
Okay. I do not insist on the change and will mark the conversation as resolved.